### ISSUE TITLE
VAMC Facility Health Services should not be limited to 3 service locations

### ISSUE THREAD

**Issue by stefaniefgray** [department-of-veterans-affairs/va.gov-cms#7252]
## Description
**User Story:** As an Editor updating the VAMC Facility Health Service page for Primary Care at the Richard L. Roudebush VAMC (VA Indiana), there are 4 Primary Care clinic locations that should be listed -- but I can only list 3 locations! 

There is no way to add the additional location to https://prod.cms.va.gov/node/39673/edit

Service Locations are limited to only three listings, and this cannot be modified.

## Acceptance Criteria
- [ ] Users can add additional service location addresses to VAMC Facility Health Services
- [ ] An editor should be able to add an additional service location to https://prod.cms.va.gov/node/39673/edit

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [x] `Platform CMS Team`
- [x] `Sitewide CMS Team`

**Comment by stefaniefgray** [comment]
Resurrecting after receiving editor question about this topic in https://va-gov.atlassian.net/browse/VAHELP-6098

cc: @davidmpickett

**Comment by mmiddaugh** [comment]
For context, while researching the number of service locations for another area of work, I did an audit and discovered there are 3 facilities with 3 service locations for primary care:

- Donald J. Mitchell Department of Veterans Affairs Outpatient Clinic
- Richard L. Roudebush Veterans & Administration Medical Center
- Schenectady VA Clinic

**Comment by mmiddaugh** [comment]
Additional use case: Mobile clinics (VHA)

There isn't a good way to represent the multiple places that a mobile clinic is available. We know this needs to be revisited in the future but hasn't hit the top of the priority list yet. 

While investigating this situation, I started to add the "stops" for a mobile unit as service delivery locations but quickly realized that being limited to 3 locations makes this approach impossible. 
- Example: [Corpus Christi Mobile](https://www.va.gov/texas-valley-health-care/locations/corpus-christi-va-mobile-clinic/) has four stops - one each day of the week (as indicated in Prepare for your visit section). 
- This isn't necessarily a viable option for other mobile clinics which have significantly more stops and variations from week to week but might have been a decent stop-gap approach for this mobile unit and its sibling: [McAllen Mobile](https://www.va.gov/texas-valley-health-care/locations/mcallen-va-mobile-clinic)

**Comment by xiongjaneg** [comment]
@mmiddaugh For the mobile clinic use case, allowing more than 3 service locations is doable, but the engineers pointed out that if a mobile clinic offers multiple services (primary, mammogram, etc.), the editor might need to add multiple service locations under each of those services. If a mobile clinic is at a different service location each weekday and it offers seven services, for example, that might be 35 service locations that the editor has to create an maintain and keep consistent. 

Any concerns about this?

**Comment by davidmpickett** [comment]
> @mmiddaugh For the mobile clinic use case, allowing more than 3 service locations is doable, but the engineers pointed out that if a mobile clinic offers multiple services (primary, mammogram, etc.), the editor might need to add multiple service locations under each of those services. If a mobile clinic is at a different service location each weekday and it offers seven services, for example, that might be 35 service locations that the editor has to create an maintain and keep consistent.
> 
> Any concerns about this?

I have been percolating an idea related to this specific multiplication problem for a while. We might want to consider creating Reusable Service Locations. The same proliferation of Service Locations could happen as we introduce Service Modalities since different Modalities of the same service may need different sets of hours and therefore different Service Locations.

**Comment by JonathanKatora** [comment]
@r-bartlett-gsa  I do not believe we should do this work under service accordions. Michelle's examples of current work arounds validate the need to provide a better solution. However, showing up in a service accordion may not be the right answer to this. Do we want to consider an EPIC to cover mobile facilities in general? I think this work would/should be prioritized under that effort.

**Comment by r-bartlett-gsa** [comment]
@JonathanKatora I have so many questions about this. In reviewing the back-end and the front-end, things don't just connect for me why service locations are listed like that. Let's leave this one open here for further investigation.
